_sass/scale.scss

```diff
@@ -110,3 +110,22 @@ nav {
     font-size: .6em;
   }
 }
+
+/* Facebook button (news page & contact page) */
+.btn-fb {
+  font-size: 0.9rem;
+
+  /* responsive displays */
+  @media #{$mq-max-xsmall} {
+    font-size: 1rem !important;
```

Comment:
Wait a minute, can't really delete the css since it does not ***only*** affect the fb button but all links on the page.